admin_manual/enterprise_user_management/user_auth_shibboleth.rst

```diff
@@ -278,6 +278,25 @@ The session timeout can be overridden in the service provider, but this
 requires a source code change of the Apache Shibboleth module. A patch can be 
 provided by the ownCloud support team.
 
+UID Considerations and windows_network_drive compatability
+----------------------------------------------------------
+
+When using user_shibboleth in SSO only mode together with user_ldap both apps need to resolve to the same uid. user_shibboleth will do the authentication and user_ldap will provide user details such as email and displayname. In the case of Active Directory multiple attributes can be used as the uid. But they all have different implications to take into account:
+
++------------+-------------------------+------------------------------+--------------------+
+| attribute  | example                 | uniqueness                   | other implications |
++------------+-------------------------+------------------------------+--------------------+
+| `sAMAccountName` | `jfd` | domain local, might change on eg. marriage | works with windows_network_drive app |
```

Comment:
Tables doesn't work in that way in rst. Also double backticks are needed in rst.

@carlaschroder to the help :)